tools/runfiles/runfiles.rs

```diff
@@ -37,6 +37,17 @@ use std::io;
 use std::path::Path;
 use std::path::PathBuf;
 
+
+/// The common name of Bazel runfiles directory suffix
+const RUNFILES_SUFFIX: &'static str = ".runfiles";
+
+/// Tests are wrapped in a `.runner` executable so we need to 
+/// also know to look for this directory as well
+#[cfg(target_family = "unix")]
+const TEST_RUNFILES_SUFFIX: &'static str = ".runner.runfiles";
```

Comment:
Could you elaborate on why do we need to look into the runners runfiles from the test?

Reply:
It's not needed anymore. I've removed this,thanks for catching it 😄